app/main.py

```diff
@@ -2,0 +1,31 @@
+def copy_file(command: str) -> None:
+    command_parts = command.split(" ")
+
+    if len(command_parts) != 3 or command_parts[0] != "cp":
+        raise ValueError(
+            "Invalid command format. "
+            "Expected format: 'cp <source_file> <destination_file>'"
+        )
+
+    source_file_name, destination_file_name = (
+        command_parts[1], command_parts[2]
```

Comment:
The variable names `source_file_name` and `destination_file_name` are descriptive and correct, which aligns with the checklist requirements.